### PR TITLE
[docs]: Updated alert & keyboard doc

### PR DESCRIPTION
**Pull request by harsha509**
## Alert

1. Added ruby sample code for alerts
2. Modified JavaScript code with correct link text

## keyBoard

1. Added ruby sample code for keyboard usage
2. Modified JavaScript sample code to target 'firefox' browser

Signed-off-by: Sri Harsha <sri_harsha509@hotmail.com>

### CODE REVIEW

**Review by diemol — CHANGES_REQUESTED** [review]
Hi @Harsha509, could you please add the changes also to the recently added Japanese language file structure?

**Comment by harsha509** [comment]
Tashikani @diemol (Sure in japanese :))

**Comment by harsha509** [comment]
Hi @diemol , i have done the requested changes.

File: docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.en.md [hunk 1]
```diff
@@ -59,7 +59,17 @@ string text = alert.Text;
 alert.Accept();
   {{< / code-panel >}}
   {{< code-panel language="ruby" >}}
-# We don't have a Ruby code sample yet -  Help us out and raise a PR  
+# Click the link to activate the alert
+driver.find_element(:link_text, 'See an example alert').click
+
+# Store the alert reference in a variable
+alert =  driver.switch_to.alert
```
Comment (diemol) [inline docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.en.md]:
```suggestion
alert = driver.switch_to.alert
```

File: docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.en.md [hunk 2]
```diff
@@ -139,11 +149,21 @@ string text = alert.Text;
 alert.Dismiss();
   {{< / code-panel >}}
   {{< code-panel language="ruby" >}}
-# We don't have a Ruby code sample yet -  Help us out and raise a PR  
+# Click the link to activate the alert
+driver.find_element(:link_text, 'See a sample confirm').click
+
+# Store the alert reference in a variable
+alert =  driver.switch_to.alert
```
Comment (diemol) [inline docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.en.md]:
```suggestion
alert = driver.switch_to.alert
```

File: docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.en.md [hunk 3]
```diff
@@ -215,11 +235,21 @@ alert.SendKeys("Selenium");
 alert.Accept();
   {{< / code-panel >}}
   {{< code-panel language="ruby" >}}
-# We don't have a Ruby code sample yet -  Help us out and raise a PR  
+# Click the link to activate the alert
+driver.find_element(:link_text, 'See a sample prompt').click
+
+# Store the alert reference in a variable
+alert =  driver.switch_to.alert
```
Comment (diemol) [inline docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.en.md]:
```suggestion
alert = driver.switch_to.alert
```

File: docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.es.md [hunk 4]
```diff
@@ -65,7 +65,17 @@ string text = alert.Text;
 alert.Accept();
   {{< / code-panel >}}
   {{< code-panel language="ruby" >}}
-# We don't have a Ruby code sample yet -  Help us out and raise a PR  
+# Click the link to activate the alert
+driver.find_element(:link_text, 'See an example alert').click
+
+# Store the alert reference in a variable
+alert =  driver.switch_to.alert
```
Comment (diemol) [inline docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.es.md]:
```suggestion
alert = driver.switch_to.alert
```

File: docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.es.md [hunk 5]
```diff
@@ -145,11 +155,21 @@ string text = alert.Text;
 alert.Dismiss();
   {{< / code-panel >}}
   {{< code-panel language="ruby" >}}
-# We don't have a Ruby code sample yet -  Help us out and raise a PR  
+# Click the link to activate the alert
+driver.find_element(:link_text, 'See a sample confirm').click
+
+# Store the alert reference in a variable
+alert =  driver.switch_to.alert
```
Comment (diemol) [inline docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.es.md]:
```suggestion
alert = driver.switch_to.alert
```

File: docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.es.md [hunk 6]
```diff
@@ -221,11 +241,21 @@ alert.SendKeys("Selenium");
 alert.Accept();
   {{< / code-panel >}}
   {{< code-panel language="ruby" >}}
-# We don't have a Ruby code sample yet -  Help us out and raise a PR  
+# Click the link to activate the alert
+driver.find_element(:link_text, 'See a sample prompt').click
+
+# Store the alert reference in a variable
+alert =  driver.switch_to.alert
```
Comment (diemol) [inline docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.es.md]:
```suggestion
alert = driver.switch_to.alert
```

File: docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.fr.md [hunk 7]
```diff
@@ -65,7 +65,17 @@ string text = alert.Text;
 alert.Accept();
   {{< / code-panel >}}
   {{< code-panel language="ruby" >}}
-# We don't have a Ruby code sample yet -  Help us out and raise a PR  
+# Click the link to activate the alert
+driver.find_element(:link_text, 'See an example alert').click
+
+# Store the alert reference in a variable
+alert =  driver.switch_to.alert
```
Comment (diemol) [inline docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.fr.md]:
```suggestion
alert = driver.switch_to.alert
```

File: docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.fr.md [hunk 8]
```diff
@@ -145,11 +155,21 @@ string text = alert.Text;
 alert.Dismiss();
   {{< / code-panel >}}
   {{< code-panel language="ruby" >}}
-# We don't have a Ruby code sample yet -  Help us out and raise a PR  
+# Click the link to activate the alert
+driver.find_element(:link_text, 'See a sample confirm').click
+
+# Store the alert reference in a variable
+alert =  driver.switch_to.alert
```
Comment (diemol) [inline docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.fr.md]:
```suggestion
alert = driver.switch_to.alert
```

File: docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.fr.md [hunk 9]
```diff
@@ -221,11 +241,21 @@ alert.SendKeys("Selenium");
 alert.Accept();
   {{< / code-panel >}}
   {{< code-panel language="ruby" >}}
-# We don't have a Ruby code sample yet -  Help us out and raise a PR  
+# Click the link to activate the alert
+driver.find_element(:link_text, 'See a sample prompt').click
+
+# Store the alert reference in a variable
+alert =  driver.switch_to.alert
```
Comment (diemol) [inline docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.fr.md]:
```suggestion
alert = driver.switch_to.alert
```

File: docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.ja.md [hunk 10]
```diff
@@ -64,7 +64,17 @@ string text = alert.Text;
 alert.Accept();
   {{< / code-panel >}}
   {{< code-panel language="ruby" >}}
-# We don't have a Ruby code sample yet -  Help us out and raise a PR  
+# Click the link to activate the alert
+driver.find_element(:link_text, 'See an example alert').click
+
+# Store the alert reference in a variable
+alert =  driver.switch_to.alert
```
Comment (diemol) [inline docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.ja.md]:
```suggestion
alert = driver.switch_to.alert
```

File: docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.ja.md [hunk 11]
```diff
@@ -144,11 +154,21 @@ string text = alert.Text;
 alert.Dismiss();
   {{< / code-panel >}}
   {{< code-panel language="ruby" >}}
-# We don't have a Ruby code sample yet -  Help us out and raise a PR  
+# Click the link to activate the alert
+driver.find_element(:link_text, 'See a sample confirm').click
+
+# Store the alert reference in a variable
+alert =  driver.switch_to.alert
```
Comment (diemol) [inline docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.ja.md]:
```suggestion
alert = driver.switch_to.alert
```

File: docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.ja.md [hunk 12]
```diff
@@ -220,11 +240,21 @@ alert.SendKeys("Selenium");
 alert.Accept();
   {{< / code-panel >}}
   {{< code-panel language="ruby" >}}
-# We don't have a Ruby code sample yet -  Help us out and raise a PR  
+# Click the link to activate the alert
+driver.find_element(:link_text, 'See a sample prompt').click
+
+# Store the alert reference in a variable
+alert =  driver.switch_to.alert
```
Comment (diemol) [inline docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.ja.md]:
```suggestion
alert = driver.switch_to.alert
```

File: docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.nl.md [hunk 13]
```diff
@@ -66,7 +66,17 @@ string text = alert.Text;
 alert.Accept();
   {{< / code-panel >}}
   {{< code-panel language="ruby" >}}
-# We don't have a Ruby code sample yet -  Help us out and raise a PR  
+# Click the link to activate the alert
+driver.find_element(:link_text, 'See an example alert').click
+
+# Store the alert reference in a variable
+alert =  driver.switch_to.alert
```
Comment (diemol) [inline docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.nl.md]:
```suggestion
alert = driver.switch_to.alert
```

File: docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.nl.md [hunk 14]
```diff
@@ -146,11 +156,21 @@ string text = alert.Text;
 alert.Dismiss();
   {{< / code-panel >}}
   {{< code-panel language="ruby" >}}
-# We don't have a Ruby code sample yet -  Help us out and raise a PR  
+# Click the link to activate the alert
+driver.find_element(:link_text, 'See a sample confirm').click
+
+# Store the alert reference in a variable
+alert =  driver.switch_to.alert
```
Comment (diemol) [inline docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.nl.md]:
```suggestion
alert = driver.switch_to.alert
```

File: docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.nl.md [hunk 15]
```diff
@@ -222,11 +242,21 @@ alert.SendKeys("Selenium");
 alert.Accept();
   {{< / code-panel >}}
   {{< code-panel language="ruby" >}}
-# We don't have a Ruby code sample yet -  Help us out and raise a PR  
+# Click the link to activate the alert
+driver.find_element(:link_text, 'See a sample prompt').click
+
+# Store the alert reference in a variable
+alert =  driver.switch_to.alert
```
Comment (diemol) [inline docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.nl.md]:
```suggestion
alert = driver.switch_to.alert
```

File: docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.zh-cn.md [hunk 16]
```diff
@@ -64,7 +64,17 @@ string text = alert.Text;
 alert.Accept();
   {{< / code-panel >}}
   {{< code-panel language="ruby" >}}
-# We don't have a Ruby code sample yet -  Help us out and raise a PR
+# Click the link to activate the alert
+driver.find_element(:link_text, 'See an example alert').click
+
+# Store the alert reference in a variable
+alert =  driver.switch_to.alert
```
Comment (diemol) [inline docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.zh-cn.md]:
```suggestion
alert = driver.switch_to.alert
```

File: docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.zh-cn.md [hunk 17]
```diff
@@ -144,11 +154,21 @@ string text = alert.Text;
 alert.Dismiss();
   {{< / code-panel >}}
   {{< code-panel language="ruby" >}}
-# We don't have a Ruby code sample yet -  Help us out and raise a PR
+# Click the link to activate the alert
+driver.find_element(:link_text, 'See a sample confirm').click
+
+# Store the alert reference in a variable
+alert =  driver.switch_to.alert
```
Comment (diemol) [inline docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.zh-cn.md]:
```suggestion
alert = driver.switch_to.alert
```

File: docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.zh-cn.md [hunk 18]
```diff
@@ -220,11 +240,21 @@ alert.SendKeys("Selenium");
 alert.Accept();
   {{< / code-panel >}}
   {{< code-panel language="ruby" >}}
-# We don't have a Ruby code sample yet -  Help us out and raise a PR
+# Click the link to activate the alert
+driver.find_element(:link_text, 'See a sample prompt').click
+
+# Store the alert reference in a variable
+alert =  driver.switch_to.alert
```
Comment (diemol) [inline docs_source_files/content/webdriver/js_alerts_prompts_and_confirmations.zh-cn.md]:
```suggestion
alert = driver.switch_to.alert
```

**Review by diemol — CHANGES_REQUESTED** [review]
Thanks for makeing the changes @Harsha509, just one last thing that I found, I made suggestions.

**Comment by harsha509** [comment]
@diemol , Updated the change. removed the extra space

**Review by diemol — APPROVED** [review]
Thank you @Harsha509!

**Pull request merged**